.travis.yml

```diff
@@ -24,19 +18,31 @@ addons:
 before_install:
   - export CXX="g++-4.8" CC="gcc-4.8"
   - nvm install 6
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
+  - 'export DISPLAY=:99.0'
+  - >-
+    /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid
+    --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0
+    1280x1024x16 -extension RANDR
   - sh -e /etc/init.d/xvfb start
 install:
- - make travissetup
+  - make travissetup
+  - pip install pygments lxml setuptools --upgrade
+  - 'git clone https://github.com/tabatkins/bikeshed.git'
+  - pip install --editable $PWD/bikeshed
+  - bikeshed update
 script:
- - make check
-
+  - bash ./.travis-deploy.sh
```

Comment:
This isn't going to work for pull requests on branches.  You should only run this if $GH_TOKEN is set.